src/WebJobs.Extensions.DurableTask/EndToEndTraceHelper.cs

```diff
@@ -254,6 +254,38 @@ public static string LocalSlotName
                 "{instanceId}: Function '{functionName} ({functionType})', version '{version}' received a '{eventName}' event. State: {state}. HubName: {hubName}. AppName: {appName}. SlotName: {slotName}. ExtensionVersion: {extensionVersion}. SequenceNumber: {sequenceNumber}.",
                 instanceId, functionName, functionType, version, eventName, FunctionState.ExternalEventRaised, hubName, LocalAppName, LocalSlotName, ExtensionVersion, this.sequenceNumber++);
         }
+        public void SendMessageFailed(
+            System.Net.HttpStatusCode statusCode,
+            string reasonPhrase,
+            string instanceId,
+            string functionName,
+            FunctionType functionType,
+            string version,
+            string reason,
+            bool isReplay,
+            FunctionState functionState,
+            string hubName,
+            string appName,
+            string slotName,
+            string extensionVersion
+            )
+        {
+            this.logger.LogError(
```

Comment:
Please add a new `EtwEventSource.Instance.ExternalEventRaised` method which matches this trace.

Reply:
Thank you! We will fix it.